unreleased.md

```diff
@@ -1,5 +1,12 @@
 # Unreleased
 
+## Improvements
+
+- Added `GetRequiredValue<T>` extension to IConfiguration with default value 
+  option which returns given value when configuration value is null
```

Comment:
remove `which ...` part of the sentence. remember this is only release notes. that part was sort of documenting the function which we don't do in release notes.